internal/metadata/request_metadata_parser.go

```diff
@@ -10,7 +10,7 @@ func (p *RequestMetadataParser) Parse(requestBody jsonrpc.RequestBody) RequestMe
 	result := RequestMetadata{}
 
 	switch requestBody.Method {
-	case "eth_getBalance", "eth_getStorageAt", "eth_getTransactionCount", "eth_getCode", "eth_call", "eth_estimateGas":
+	case "eth_getBalance", "eth_getStorageAt", "eth_getTransactionCount", "eth_getCode", "eth_call", "eth_estimateGas", "trace_filter":
```

Comment:
Should we update the documentation to match the behavior change? Everything says `state` right now. We can change it to `StateAndTrace` and clarify w/ comments.

Mind also adding these trace methods? https://openethereum.github.io/JSONRPC-trace-module#:~:text=G%3A%20%5B1%2C%200%5D%7D%20%5D-,JSON%2DRPC%20methods,-Ad%2Dhoc%20Tracing